tests/smoke/templates/sar/AmazonConnectSalesforceLambda-template.yaml

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
probably not needed.

Reply:
Yes I removed that but forgot to commit it to see if windows would pass.